ac.soton.eventb.statemachines.animation.feature/feature.xml

```diff
@@ -17,23 +17,23 @@
    </copyright>
 
    <url>
-      <update label="%updateSiteName" url="http://rodin-b-sharp.sourceforge.net/updates/site.xml"/>
+      <update label="%updateSiteName" url="https://eventB-soton.github.io/updateSite/site.xml"/>
    </url>
 
```

Comment:
There is no site.xml for "eventB-soton.github.io/updateSite"

Reply:
thanks.. well spotted